### PR TITLE
add catalog source catagory

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

File: content/docs/integrations/roadie-api/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+humanName: Roadie API
+logoImage: '../../../assets/logos/roadie/roadie-racks.png'
+integrationType: Catalog Source
+---
+
+The Roadie Entity API allows you to create, update and delete entities in the Roadie catalog via the public API. This option allows you to manage entities from sources where Roadie does not have an existing Entity provider.
+
+Before you start you will first need to [get an API key](/docs/api/authorization/).
+
+You can either manage entities as a full set of entities, or create and delete entities one by one. You would use entity sets if you want to create a batch of entities and manage them on an ongoing basis in an idempotent manner. For example, you might want to run a script once a week that lists resources from your internal system and creates relevant entities in Roadie. Then when resources are added and removed, the entities are added and removed.
```
Comment (punkle) [inline content/docs/integrations/roadie-api/index.md:15]: @kissmikijr

File: src/pages/docs/integrations.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import customPluginsLogoImagePath from '../../../content/assets/logos/code/code.
 const INTEGRATION_TYPE_FILTERS = Object.freeze({
```
Comment (punkle) [inline src/pages/docs/integrations.js:10]: I feel like these categories should be "Catalog Source", "Frontend Plugin" and "Search".

File: content/docs/integrations/http/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+title: HTTP
+publishedDate: '2024-02-20T21:00:00.0Z'
+description: How to manage entities via a HTTP URL
+
+humanName: HTTP
+logoImage: '../../../assets/logos/http/http-logo.png'
+integrationType: Catalog Source
+---
+
+You may choose to load entities from a http server hosted on the public internet. In order to enable this, you will need to tell Roadie in advance the allowed URLs.
```
Comment (Xantier) [inline content/docs/integrations/http/index.md:11]:
```suggestion
You may choose to load entities from an HTTP server hosted on the public Internet. In order to enable this, you will need to tell Roadie in advance the allowed URLs.
```

File: content/docs/integrations/http/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Steps
+### Host an entity yaml file
+You will need to host a yaml file with the correct Backstage entity schema on a http server that is available to Roadie.
```
Comment (Xantier) [inline content/docs/integrations/http/index.md:15]:
```suggestion
You will need to host a YAML file with the correct Backstage entity schema on a HTTP server that is available to Roadie.
```

**Review by Xantier — APPROVED** [review]

File: content/docs/integrations/roadie-api/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        "spec": {
+          "type": "library",
+          "owner": "roadie-demo/dev-team",
+          "lifecycle": "experimental",
+          "system": "core"
+        }
+      }'
+```
+
+You can retrieve an entity with the entity id.
```
Comment (kissmikijr) [inline content/docs/integrations/roadie-api/index.md:96]: How do I get this entity ID ? Is on the UI when I inspect my entity that I added to my catalog?

File: content/docs/integrations/roadie-api/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kissmikijr) [reply content/docs/integrations/roadie-api/index.md:15]:
```suggestion
You can either manage entities as a full set of entities, or create and delete entities one by one. 
You would use entity sets if you don't want to manually manage the creation and deletion and figure out the diff between your existing entities.  You would like to manage them on an ongoing basis in an idempotent manner. For example, you might want to run a script once a week that lists resources from your internal system and creates relevant entities in Roadie. Then when resources are added and removed, the entities are added and removed.
```

**Pull request merged**